library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -1101,4 +1102,16 @@ private int translateStatus(int status) {
             }
         }
     }
+
+    /**
+     * Restart will activate the downloads workflow in case that it was not already active
+     * <p/>
+     * A possible scenario: A client denies a download for a particular business rule and that
+     * rule does not apply any more. Calling this method will reactivate the downloads workflow,
+     * check the client rules and proceed if necessary
+     */
+    public void restart() {
```

Comment:
oh yes that makes sense :+1:  restart sounds so brutal lol `prod()` :=)  

![](http://www.syrvet.com/sitevizenterprise/website/cgi-bin/file.pl/syrvet/media/products/handw_fullprod_web_BLACK_7CC2A2595FC85.gif?dimensions=460x350)